### PR TITLE
Updating Label H1 Preamble POS for groundspeed.

### PR DESCRIPTION
**Pull request by johnsom**

### CODE REVIEW

**Review by kevinelliott — APPROVED** [review]

File: research/H1/POS.md [hunk 1]
```diff
@@ -56,14 +58,18 @@ Flight was from Washington, DC (IAD) to Fort Lauderdale, FL (FLL).
...
 - TS - Time Stamp
 - W - Longitude West
+- P - Plus Celcius
+- M - Minus Celcius
 
 ## Analysis
 
 Comma separated list:
-\<Position\>,\<Future WayPoint\>,\<Unknown\>,\<Altitude in hundreds\>,\<Future WayPoint\>,\<Unknown\>,\<Future WayPoint\>,\<Unknown\>,\<Unknown\>,\<Unknown/TimeStamp\>,\<Unknown/Date\>,\<Unknown\>,\<Unknown\>,\<Unknown\>
+\<Position\>,\<Future WayPoint\>,\<Unknown\>,\<Altitude in hundreds\>,\<Future WayPoint\>,\<Unknown\>,\<Future WayPoint\>,\<Outside Air Temperature\>,\<Unknown\>,\<Unknown/TimeStamp\>,\<Groundspped or Date\>,\<Unknown\>,\<Unknown\>,\<Unknown\>
```
[the remark this answers is not in the document]
Comment (kevinelliott) [reply research/H1/POS.md:67]: Hey @johnsom just touching base on this. I can adjust if you can't, so we can get this merged in.

[the remark this answers is not in the document]
Comment (johnsom) [reply research/H1/POS.md:67]: I did not see that. I can fix that this weekend

[the remark this answers is not in the document]
Comment (johnsom) [reply research/H1/POS.md]: The magic of the edit button. Done

**Comment by kevinelliott** [comment]
Awesome! Thanks @johnsom

**Pull request merged**